src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -103,7 +104,9 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
         );
       }
       break;
-
+    case VenueTemplate.externalroom:
```

Comment:
we don't need a new venueTemplate type. It should replace `zoomroom/artcar/etc`

Reply:
fixed in 733405113371f979b4941af7d375adc1f1e3678c